lib/matplotlib/colors.py

```diff
@@ -37,27 +37,25 @@
 Matplotlib recognizes the following formats to specify a color:
 
 * an RGB or RGBA tuple of float values in ``[0, 1]`` (e.g., ``(0.1, 0.2, 0.5)``
-  or  ``(0.1, 0.2, 0.5, 0.3)``);
-* a hex RGB or RGBA string (e.g., ``'#0F0F0F'`` or ``'#0F0F0F0F'``);
+  or ``(0.1, 0.2, 0.5, 0.3)``);
+* a hex RGB or RGBA string (e.g., ``'#0F0F0F'`` or ``'#0f0f0f0f'``);
```

Comment:
```suggestion
* a hex RGB or RGBA string (e.g., ``'#0f0f0f'`` or ``'#0f0f0f80'``);
```
- I would not change the case when actually showcasing the difference between RGB and RGBA. That's distracting from the original purpose.
- I recommend to use a different value for alpha than the RGB values. While #0f0f0f is common it's unlikely that the alpha value would be the same, this makes the example more realistic and easier to understand. You may add an additional note that the string is case insensitive.
- [minor] I have a slight preference for lowercase hex color values. HTML5 specifies "valid simple color" and the specialization [valid lowercase simple color](https://www.w3.org/TR/2011/WD-html5-20110525/common-microsyntaxes.html#valid-lowercase-simple-color) - but not an uppercase simple color. This indicates that lowercase is more important, also it's the standard for serializing color values in HTML5. Another hint is that [some random thread on RGB case](https://robertnyman.com/2006/10/23/upper-or-lowercase-hex-codes-in-css/) mentions the lowercase variant twice as often.

Reply:
done